fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/config/FineractProperties.java

```diff
@@ -36,6 +37,8 @@ public class FineractProperties {
 
     private FineractCorrelationProperties correlation;
 
+    private FineractPartitionedJobProperties fineractPartitionedJobProperties;
```

Comment:
I think the "fineract" prefix is not necessary here; this nested class is already in FineractProperties... so kind of redundant. Same for the suffix "Properties"; that kind of is self-evident.
I'd suggest to name this attribute just `partionedJob` if you insist using the class `FineractPartitionedJobProperties`; but as this class only contains 1 attribute I'd just move `List<JobProperty> jobProperties` here and name the variable `partitionedJobProperties`